frontend/src/index.css

```diff
@@ -21,3 +23,20 @@ code {
   border: none;
   z-index: 10;
 }
+
+.search {
+  position: absolute;
+  top: 1rem;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 100%;
+  max-width: 400px;
+  z-index: 10;
+}
+
+.search input {
+  padding: 0.5rem;
+  font-size: 1.5rem;
+  width: 100%;
+  text-align: center;
+}
```

Comment:
I think this belongs inside a component, not a global css file. Think about using styled-components and putting it in `GlobalStyle`